c++/src/kj/compat/tls.c++

```diff
@@ -905,7 +905,7 @@ TlsCertificate::TlsCertificate(kj::ArrayPtr<const kj::ArrayPtr<const byte>> asn1
   for (auto i: kj::indices(asn1)) {
     auto p = asn1[i].begin();
 
-    // "_AUX" apparently refers to some auxilliary information that can be appended to the
+    // "_AUX" apparently refers to some auxillary information that can be appended to the
```

Comment:
I think `auxiliary` is actually the common spelling. (And again, below.)

(Nice job!)

Reply:
Done :)